packages/mui-material/test/typescript/hoc-interop.spec.tsx

```diff
@@ -21,23 +21,23 @@ const filledProps = {
 
 // baseline behavior
 <TextField variant="filled" {...filledProps} />;
-// @ts-expect-error
-<TextField {...filledProps} />; // desired to throw
```

Comment:
I think it should be renamed to `outlinedProps`, because variant is not provided, and default is outlined variant. Same applied below. What do you think?

Reply:
Or maybe have two constants filledProps and outlinedProps

Reply:
agree with adding new variable. resolved in this commit https://github.com/mui/material-ui/pull/36737/commits/9e9cb3d1b2e6571564e15702a5f1f12cb1e182e4